acceptance-tests/src/test/java/uk/gov/di/test/acceptance/LoginStepDefinitions.java

```diff
@@ -15,10 +15,7 @@
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
-import static uk.gov.di.test.acceptance.AuthenticationJourneyPages.ENTER_CODE;
-import static uk.gov.di.test.acceptance.AuthenticationJourneyPages.ENTER_EMAIL_EXISTING_USER;
-import static uk.gov.di.test.acceptance.AuthenticationJourneyPages.ENTER_PASSWORD;
-import static uk.gov.di.test.acceptance.AuthenticationJourneyPages.SIGN_IN_OR_CREATE;
+import static uk.gov.di.test.acceptance.AuthenticationJourneyPages.*;
```

Comment:
there's an ide setting to turn off * imports

Reply:
Done, thanks :)